docs/data/data-grid/joy-ui/GridJoyUISlots.tsx

```diff
@@ -10,8 +11,64 @@ import {
 } from '@mui/material/styles';
 import { CssVarsProvider as JoyCssVarsProvider } from '@mui/joy/styles';
 import type {} from '@mui/material/themeCssVarsAugmentation';
+import DeleteIcon from '@mui/icons-material/Delete';
 
-const materialTheme = materialExtendTheme();
+const materialTheme = materialExtendTheme({
+  components: {
+    MuiSvgIcon: {
+      styleOverrides: {
+        root: ({ ownerState }) => {
+          return {
+            color: 'var(--Icon-color)',
+            margin: 'var(--Icon-margin)',
+            ...(ownerState.fontSize &&
+              ownerState.fontSize !== 'inherit' && {
+                fontSize: `var(--Icon-fontSize, 1.5rem)`,
+              }),
+          };
+        },
+      },
+    },
+  },
+});
+
+const columns: GridColDef[] = [
```

Comment:
That's odd, I still see the old dataset on a page preview: https://deploy-preview-8699--material-ui-x.netlify.app/x/react-data-grid/experiments/joy-ui/

I'll push a commit with `docs:typescript:formatted` and see if it fixes the issue